Gemfile

```diff
@@ -13,6 +13,8 @@ gem 'active_hash'
 gem 'sanitize'
 gem 'gmaps4rails'
 gem 'geocoder'
+gem 'omniauth-google-oauth2'
+gem 'omniauth-oauth2', '~> 1.3.1'
```

Comment:
The explicit dependency on `omniauth-oauth2` shouldn't be needed, since it's a dependency of `omniauth-google-oauth2` which explicitly locks it to 1.3.1 (see https://github.com/zquestz/omniauth-google-oauth2/commit/fc170504bf8273ec6601bf8ad75d98bf14260976)

Probably can fix that by removing line 17 and running `bundle update omniauth-google-oauth2`